lk2nd/device/dts/msm8952/msm8937-nokia-nd1.dts

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include <skeleton64.dtsi>
+#include <lk2nd.dtsi>
+
+/ {
+	qcom,msm-id = <QCOM_ID_MSM8937 0x2000>;
+	qcom,board-id = <0x94 0>;
+};
+
+&lk2nd {
+    model = "HMD Global Nokia 5 (nd1)";
+    compatible = "nokia,nd1";
+    lk2nd,match-device = "ND1";
+    lk2nd,match-panel;
```

Comment:
```suggestion

```

No need for match properties if youre doing top-level node. Can maybe keep match-device for documentation, match-panel is redundant in either case.